### DISCUSSION TITLE
Loop current chapter

### DISCUSSION THREAD

**Discussion by SameenAhnaf** [libre-tube/LibreTube#5164] [Ideas]
### Describe your suggested feature

Add the ability to loop only current chapter from Chapters menu. If this option is enabled, a warning like `A chapter is already looped. How to proceed?` if play queue loop option is clicked on. The entire part after the highlight of the video should be looped in case of Sponsorblock highlight. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
> Add the ability to loop only current video from play queue.
>
That's already supported in the latest debug builds.

**Comment by Bnyro** [comment]
Converting to a discussion as I don't really see how this would be useful for the average user.
